WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -347,6 +351,23 @@ public static boolean supportsLayoutGridFeature(SiteModel site) {
         return site != null && (site.isWPCom() || site.isWPComAtomic());
     }
 
+    public static boolean supportsFacebookEmbedFeature(SiteModel site) {
+        return site != null && (site.isWPCom() || checkMinimalJetpackVersion(site, WP_FACEBOOK_EMBED_JETPACK_VERSION));
+    }
+
+    public static boolean supportsInstagramEmbedFeature(SiteModel site) {
+        return site != null && (site.isWPCom() || checkMinimalJetpackVersion(site, WP_INSTAGRAM_EMBED_JETPACK_VERSION));
+    }
+
+    public static boolean supportsLoomEmbedFeature(SiteModel site) {
+        return site != null && (site.isWPCom() || checkMinimalJetpackVersion(site, WP_LOOM_EMBED_JETPACK_VERSION));
+    }
+
+    public static boolean supportsSmartframeEmbedFeature(SiteModel site) {
+        return site != null && (site.isWPCom() || checkMinimalJetpackVersion(site,
+                WP_SMARTFRAME_EMBED_JETPACK_VERSION));
+    }
```

Comment:
@fluiddot I am seeing here where we are repeating the logic within these functions. How about we extract it to a function and then pass in the flag itself? Let me know.

Reply:
Yeah good idea, I'll unify this code, I totally agree that is quite redundant 👍 .

Reply:
I've applied your suggestion in [this commit](https://github.com/wordpress-mobile/WordPress-Android/pull/15378/commits/395156cc4c02f38fbd244c3c14a9f03c398376d7).